File: komock-core/src/main/kotlin/ua/com/lavi/komock/engine/handler/response/EmptyResponseHandler.java
package ua.com.lavi.komock.engine.handler.response;

import org.jetbrains.annotations.NotNull;
import ua.com.lavi.komock.engine.model.Request;
import ua.com.lavi.komock.engine.model.Response;

public class EmptyResponseHandler implements ResponseHandler {

    @Override
    public void handle(@NotNull Request request, @NotNull Response response) {
        // nothing to do
    }
}

File: komock-core/src/main/kotlin/ua/com/lavi/komock/engine/handler/response/EmptyResponseHandler.kt
package ua.com.lavi.komock.engine.handler.response

import ua.com.lavi.komock.engine.model.Request
import ua.com.lavi.komock.engine.model.Response

/**
 * Created by Oleksandr Loushkin
 */

class EmptyResponseHandler : ResponseHandler {

    override fun handle(request: Request, response: Response) {
        // nothing to do
    }
}
